Homework/css/styles.css

```diff
@@ -0,0 +1,95 @@
+
+body {	
+
```

Comment:
:hammer: Spacing! I would strongly suggest getting into the habit of both indenting and spacing correctly. It's best practice. In production, you'll be looking at well-organized code that is visually understandable. Extra empty lines & poor indentation make that difficult. Your indents look fine, but there are extra lines throughout this file (and others).